src/main/java/io/vertx/core/http/HttpClientOptions.java

```diff
@@ -94,6 +94,11 @@
   public static final int DEFAULT_MAX_WEBSOCKET_FRAME_SIZE = 65536;
 
   /**
+   * The default value for maximum websocket messages (could be assembled from multiple frames) = 655360 bytes
+   */
+  public static final int DEFAULT_MAX_WEBSOCKET_MESSAGE_SIZE = 655360;
```

Comment:
I'm wondering if we should not a lower value, what are the usual cases for this feature ?